wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationTableViewDataSource.swift

```diff
@@ -480,8 +480,17 @@ extension ConversationTableViewDataSource {
         // 45 minutes
         let significantTimeInterval: TimeInterval = 60 * 45
         let isTimeIntervalSinceLastMessageSignificant: Bool
+
+        let isTimestampInSameMinuteAsPreviousMessage: Bool
+
         let previousMessage = messagePrevious(to: message, at: index)
 
+        if let currentMessage = message.serverTimestamp, let prevMessage = previousMessage?.serverTimestamp {
+            isTimestampInSameMinuteAsPreviousMessage = currentMessage.isInSameMinute(asDate: prevMessage)
```

Comment:
Maybe a less specific name would fit better here.

`isMessageCloseToPreviousOne` or
`isProximateMessage`

Reply:
I think in this case the more specific the better. Based on how it's being used.